diff -ur efingerd-1.6/Makefile efingerd-1.6.new/Makefile
--- efingerd-1.6/Makefile	Mon Mar 12 14:34:49 2001
+++ efingerd-1.6.new/Makefile	Tue Oct  9 17:56:13 2001
@@ -1,15 +1,15 @@
 
-BINDIR=/usr/local/sbin
-MANDIR=/usr/local/man/man8
+BINDIR=$(DESTDIR)/usr/sbin
+MANDIR=$(DESTDIR)/usr/share/man/man8
 
 CC = gcc 
-CFLAGS = -O2 -Wall -Wsurprising
+#CFLAGS = -O2 -Wall -Wsurprising
 
 #or, if you do not have libinent at all:
 #CFLAGS = -O2 -Wall -DDONT_HAVE_LIBIDENT
 
 #uncomment the following line if you do have libident.so
-#LDFLAGS = -lident
+LDFLAGS = -lident
 
 
 all: efingerd
@@ -19,8 +19,7 @@
 #replace the line below with the line above if you have libident.so, or 
 #do not have libident at all
 #modify path to libident.a if necessary
-	$(CC) $(CFLAGS) $(LDFLAGS) efingerd.o child.o /usr/lib/libident.a -o efingerd
-	strip efingerd
+	$(CC) $(CFLAGS) $(LDFLAGS) efingerd.o child.o -o efingerd
 
 efingerd.o: efingerd.c
 	$(CC) $(CFLAGS) -c efingerd.c
@@ -32,9 +31,11 @@
 	rm -f *~ *.o efingerd
 
 install: efingerd 
+	mkdir -p $(BINDIR)
 	cp efingerd $(BINDIR)
 	mkdir -p $(DESTDIR)/etc/efingerd
 	cp examples-standard/* $(DESTDIR)/etc/efingerd
 
 install-doc: efingerd 
-	gzip -9 efingerd.8 -c >$(MANDIR)/efingerd.8.gz
+	mkdir -p $(MANDIR)
+	cp efingerd.8 $(MANDIR)/efingerd.8
Only in efingerd-1.6.new/: Makefile.orig
Only in efingerd-1.6.new/: Makefile.rej
